pkg/apis/app/v1alpha1/zz_generated.openapi.go

```diff
@@ -50,7 +50,6 @@ func schema_pkg_apis_app_v1alpha1_Builds(ref common.ReferenceCallback) common.Op
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
 				Description: "Builds ...",
-				Type:        []string{"object"},
```

Comment:
Same question here. This file shouldn't be changed. If it's, ok, but just double check if you upgrade to 0.11.

Reply:
I didn't, was using sdk version 0.10